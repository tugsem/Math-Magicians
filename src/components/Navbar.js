import React from 'react';

const Navbar = () => {
  const links = [
    {
      id: 1,
      path: '/',
      text: 'Home',
    },
    {
      id: 2,
      path: '/calculator',
      text: 'Calculator',
    },
     {
      id: 3,
      path: '/quote',
      text: 'Quote',
    }
  ];
  return (
    <nav className="navBar">
      <h1 className='title'>Math Magicians</h1>
      <ul className='navList'>
        {links.map((link) => (
          <li key={link.id}>
            <a href={link.path}>
              {link.text}
            </a>
          </li>
        ))}
      </ul>
    </nav>
  );
};
export default Navbar;