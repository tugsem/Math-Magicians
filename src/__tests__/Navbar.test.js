import renderer  from 'react-test-renderer';
import Navbar from '../components/Navbar';

it('Renders correctly',()=>{
    const tree = renderer.create(
        <Navbar />).toJSON();
        expect(tree).toMatchSnapshot();
})