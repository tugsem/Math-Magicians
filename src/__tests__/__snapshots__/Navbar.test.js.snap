// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<nav
  className="navBar"
>
  <h1
    className="title"
  >
    Math Magicians
  </h1>
  <ul
    className="navList"
  >
    <li>
      <a
        href="/"
      >
        Home
      </a>
    </li>
    <li>
      <a
        href="/calculator"
      >
        Calculator
      </a>
    </li>
    <li>
      <a
        href="/quote"
      >
        Quote
      </a>
    </li>
  </ul>
</nav>
`;
